packages/plugin-e2e/src/models/TimeRange.ts

```diff
@@ -0,0 +1,47 @@
+import { Expect } from '@playwright/test';
+import { PluginTestCtx, TimeRangeArgs } from '../types';
+import { GrafanaPage } from './GrafanaPage';
+
+export class TimeRange extends GrafanaPage {
+  constructor(ctx: PluginTestCtx, expect: Expect<any>) {
+    super(ctx, expect);
+  }
+
+  async set({ from, to, zone }: TimeRangeArgs) {
+    try {
+      await this.ctx.page
+        .getByLabel(this.ctx.selectors.components.PanelEditor.General.content)
+        .locator(`selector=${this.ctx.selectors.components.TimePicker.openButton}`)
+        .click({ force: true, timeout: 2000 });
```

Comment:
Why do we need the `2000ms` timeout here?

Reply:
Good question...this is mostly copied from plugin/e2e in grafana core. I'll remove it and we'll see. If timeout is not provided explicitly like in this case, timeout from playwright config will be used. It defaults to 5000ms if I recall correctly. Will changed this!